atlasexec/atlas.go

```diff
@@ -303,7 +285,7 @@ func (c *Client) MigrateApply(ctx context.Context, params *MigrateApplyParams) (
 }
 
 // SchemaApply runs the 'schema apply' command.
-func (c *Client) SchemaApply(ctx context.Context, params *SchemaApplyParams) (*SchemaApply, error) {
+func (c *Client) SchemaApply(ctx context.Context, params *SchemaApplyParams) ([]*SchemaApply, error) {
```

Comment:
I prefer the principle where the default behavior should be simple and intuitive and the complex use case should work harder. 

In this case, I prefer to keep this as the default but to add another function for multiple targets, the implementation can (and should) error if multi results are returned to the single API call. 

Thoughts @a8m ?